crates/sui-bridge-indexer/src/worker.rs

```diff
@@ -164,13 +168,19 @@ pub async fn process_eth_transaction(
                             block_height: block_number,
                             timestamp_ms: timestamp,
                             txn_hash: tx_hash.as_bytes().to_vec(),
+                            sender_address: format!(
+                                "0x{}",
+                                hex::encode(bridge_event.sender_address.as_bytes()),
+                            ),
```

Comment:
@Bridgerz here and below, thoughts?
I could not get the entire address to be saved and it was always truncated so I had to go through this rather "funny" approach...

Reply:
sync'ed with @Bridgerz and we are good here